macros/staging/union/zendesk_source_relation.sql

```diff
@@ -6,7 +6,7 @@
 
 {% macro default__apply_source_relation() -%}
 
-{% if var('zendesks', []) != [] %}
+{% if var('zendesk_sources', []) != [] %}
```

Comment:
This got mistakenly switched during the source package deprecation and causes all rows to have the same source_relation